calculator.js

```diff
@@ -0,0 +1,104 @@
+let operator = "";
+let previousValue = "";
+let currentValue = "";
+
+const choices = {
+  addition: "+",
+  subtraction: "-",
+  multiplication: "x",
+  division: "/",
+  decimal: ".",
+};
+
+document.addEventListener("DOMContentLoaded", function () {
+  const clear = document.querySelector("#clear-btn");
+  const equal = document.querySelector(".equal");
+  const decimal = document.querySelector(".decimal");
+
+  const numbers = document.querySelectorAll(".number");
+  const operators = document.querySelectorAll(".operator");
+
+  const previousScreen = document.querySelector(".previous");
+  const currentScreen = document.querySelector(".current");
+
+  numbers.forEach((number) =>
+    number.addEventListener("click", function (e) {
+      handleNumber(e.target.textContent);
+      currentScreen.textContent = currentValue;
+    })
+  );
+
+  operators.forEach((op) =>
+    op.addEventListener("click", function (e) {
+      handleOperator(e.target.textContent);
+      previousScreen.textContent = previousValue + " " + operator;
+      currentScreen.textContent = currentValue;
+    })
```

Comment:
pls add a check here so that operators should not be allowed until there's a number inputted first

Reply:
Done mam.